balrog/go.sum

```diff
@@ -0,0 +1,382 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I'm wondering if all these dependencies are a problem.

Many are not, to my reading of the code, needed by any of the code paths we use, but are part of other functions of the "autograph" module which includes a variety of features.

So, if we really didn't want to use the autograph module as packaged, we could probably do something with git submodules and then copying just the pieces of code we need (the contentsignaturepki directory and maybe one other). Or just manually copy in the files, but with submodules we could more easily stay up todate. But this would definitely be a little gross.

Reply:
If those dependencies are included in the final dll/so, probably it's better to reduce them to the minimum. In this way, we reduce the size of the final package.
I also wonder if there are issues with the licenses.

How long would it take to replace the "autograph" module with a smaller package? Via submodule probably, as you suggested?